src/components/Card.js

```diff
@@ -2,14 +2,8 @@ import React from 'react';
 import styled from 'styled-components';
 
 const CardContainer = styled.div`
-  background-color: white;
-  height: 70%;
-  margin: 0 2rem;
-  overflow: auto;
-  border-width: thin;
-  border-radius: 1rem;
-  box-shadow: 0 0.7rem 1.2rem rgba(0, 0, 0, 0.09),
-    0 0.4rem 0.4rem rgba(0, 0, 0, 0.16);
+  height: 100%;
+  padding-top: 2%;
```

Comment:
Another question about using `rem` and `%` - is the idea here to use `%` for height and `rem` for the rest?

Reply:
I'm not really sure to be honest :man_shrugging:

Reply:
Do we want to style everything with one of them only? I would say that `%` makes sense for height and width, but maybe for all the rest we should use `rem` :straight_ruler:

Reply:
I think it depends on if we want things to be a set height. Eg the navbar would look fine with one percentage in mobile but in desktop the percentage would look silly so rem makes sense for this. I will have a look at what I'm styling here and see what makes sense but maybe its just dependant on the specific case

Reply:
Sure, should I wait with approving the PR or go for it?  :running_woman:

Reply:
I just deleted that line and things didn't change haha so who knows what that was for :man_shrugging:

Reply:
haha... approving :smile: